asapdiscovery/simulation/scripts/MProP0009_Sub.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#BSUB -J run_single_simulation
+#BSUB -oo log_files/prep_full_mpro_AP_test.out
+#BSUB -e log_files/prep_full_mpro_AP_test.stderr
+#BSUB -n 1
+#BSUB -R rusage[mem=4]
+#BSUB -q gpuqueue
+#BSUB -W 12:00
+mkdir -p /data/chodera/asap-datasets/prepped_mpro_P0009/
+
+source ~/.bashrc
+conda activate docking
+
+python /data/chodera/lemonsk/covid-moonshot-ml/asapdiscovery/simulation/scripts/MproP0009.py \
+
+-i /data/chodera/asap-datasets/full_frag_prepped_mpro_12_2022/Mpro-P0009_0A_MAT-POS-f2460aef-1/prepped_receptor_0.pdb \
+-o /data/chodera/lemonsk/default_out \
+
```

Comment:
the backslash is commenting out the newline, but it won't comment lines 15 and 18, so this will throw an error. remove lines 15 and 18. (the empty ones)